python/py-tensorflow-metadata/Portfile

```diff
@@ -58,35 +56,46 @@ if {${name} ne ${subport}} {
                     ${bazel_build_workspace_path}/tensorflow_metadata/proto/v0
     
     post-extract {
-        xinstall -d ${workpath}/bin ${bazel_build_proto_path}
+        xinstall -d \
+                    ${workpath}/bin \
+                    ${bazel_build_proto_path}
         ln -s ${python.bin} ${workpath}/bin/python
     }
 
     # replace with patch files after this repo is a little more stable
+    set bazel_build_opts [bazel::get_opts]
```

Comment:
`bazel::get_opts` no longer exists in the bazel PG. It is split into `bazel::get_cmd_opts` and `get_build_opts`. Please make sure you have the latest version of the PG from git, and update accordingly.

Reply:
I suspect the way it currently is in the port in git, following my last pushes is in fact correct here.

Reply:
Lol, it was working because I was using your version. Sorry, I'll fix this commit.

Reply:
P.S. I like the clever way you avoid having to redefine `build.cmd` and others so that Python building works.